.circleci/config.yml

```diff
@@ -176,18 +155,90 @@ jobs:
               ]
             }
           channel: team-mobile-bots  
+  trigger-spec-test-pipeline:
+      docker: 
+        - image: cimg/base:current
+      resource_class: small
+      steps:
+        - run:
+            name: Trigger spec pipeline
+            # If there's a spec repo branch with the same name as the one currently executing,
+            # trigger against that as a means to allow coordinated test updates.
+            # If we get `{ "message" : "Branch not found" }`, fall back to running on `main`.
+            command: |
+              REMOTE_BRANCH=$CIRCLE_BRANCH
 
-  update_code_coverage:
-    executor: android_compatible
+              create_remote_pipeline () {
+                echo "Try remote job for $REMOTE_BRANCH"
+                CREATED_PIPELINE_ID=$(curl --request POST \
+                  --url https://circleci.com/api/v2/project/github/appcues/appcues-mobile-experience-spec/pipeline \
+                  --header "Circle-Token: $CIRCLE_API_TOKEN" \
+                  --header "content-type: application/json" \
+                  --data "{ \"branch\": \"$REMOTE_BRANCH\", \"parameters\": { \"platform\": \"android\", \"sdk-branch\": \"$CIRCLE_BRANCH\", \"triggering-pipeline-id\": \"<< pipeline.id >>\" }}" \
```

Comment:
key point here setting platform `android` to trigger the right validation suite

Reply:
validation suit is setup in circle?

Reply:
that part is here: https://github.com/appcues/appcues-mobile-experience-spec/pull/218